generated/nitclk/nitclk_library.cpp

```diff
@@ -6,7 +6,7 @@
 #include "nitclk_library.h"
 
 #if defined(_MSC_VER)
-static const char* kLibraryName = "nitclk_64.dll";
+static const char* kLibraryName = "niTClk_64.dll";
```

Comment:
Note: I think this case change for the dll is OK and correct based on the base hapigen metadata.